collection-scripts/targeted_crs

```diff
@@ -135,6 +135,13 @@ if [ ! -z "${vm_resources}" ]; then
         dump_resource "virtualmachine" $target_vm_name $target_ns
         present_vm_resources+=("$target_vm_name")
 
+        if [ -z $PLAN ]; then
+          migration_id=($(echo $vm_data | jq -r '.metadata.labels.migration'))
```

Comment:
ok so you assume here that vm_data was set, which means `if [ ! -z "${vm_resources}" ]; then` so how about checking either vm_data or vm_resources is set rather than checking if the plan wasn't set?

Reply:
i don't see the benefit in such change. if we have the plan we don't need this block here, as we set already the migration ids based on the plan.

Reply:
done with option 2, i need to verify.

Reply:
yeah, I hoped you'd pick that option :)